news/2019-11-14-beatmap-pack-management-update-1.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: Introducing Beatmap Pack Management
+date: 2019-11-14 14:00:00 +0000
+---
+
+An update to the Beatmap Pack Management project, including new pack releases!
+
+![](/wiki/shared/news/banners/beatmap-packs.jpg)
+
+Our project is called Beatmap Pack Management and we are focused on development of the Beatmap Packs. Many of you might not even know who we are or what we are doing, so I'd like to present some information about us:
+
+I've created a small group of experienced users who are interested in working on handmade charts dedicated to specific themes, ideas, or beatmaps which share similarities. Since July 2019, we've been actively working to provide new packs of interesting themes. Interest in this project is very high and some noteworthy members of the community have joined the group to help us by contributing their ideas to the project. We already have a few things to present to you, and I am sure you'll like them.
```

Comment:
```suggestion
I've created a small group of experienced users interested in working on packs of beatmaps dedicated to specific themes, similarities, and ideas. We have been actively working since July 2019 to provide new and interesting packs to play! Since we began the Beatmap Pack Management initiative, interest in this project has been very high and some noteworthy members of the community have also joined the group to help us by contributing their ideas to the project. We already have a few things to present to you which I'm sure you'll like. Be sure to check out the packs below!
```